autopts/wid/bap.py

```diff
@@ -1249,7 +1249,7 @@ def get_audio_locations_from_pac(addr_type, addr, audio_dir):
 
 
 ac_configs = {
-    # test_case_name: ([CIS_1, CIS_2, ...], num_of_servers)
+    # test_case_name: ([CIS_1, CIS_2, ...], num_of_servers, mono)
```

Comment:
Would love to see this ~monstrosity~ creative solution to be refactored into a Python data class (https://docs.python.org/3/library/dataclasses.html)